@here/harp-map-theme/resources/berlin_tilezen_base.json

```diff
@@ -1932,6 +1999,51 @@
                 },
                 "renderOrder": 100
             }
+        ],
```

Comment:
Is this intended change ? (doesn't look like)

Reply:
True, I was actually stumbling upon all the slowness when enabling terrain (it is already slow w/o terrain, but w/ terrain it is even slower).
Not sure if you or @robertoraggi are looking into the styling for multiple data-sources right now, but it would be nice if we would not have to add the terrain block to every theme. What I could do for now, is to check if the theme contains a terrain block and if not, I add it on the fly (in MapViewDemo). What do you think?